puppet/modules/jenkins_job_builder/files/theforeman.org/pipelines/release/pipelines/plugins-pipeline.groovy

```diff
@@ -0,0 +1,22 @@
+pipeline {
+    agent none
+
+    options {
+        timestamps()
+        timeout(time: 2, unit: 'HOURS')
+        disableConcurrentBuilds()
+        ansiColor('xterm')
+    }
+
+    stages {
+        stage('Install Test') {
+            agent { label 'el' }
```

Comment:
It only really depends on nodes that have the secret `centos-jenkins`

Reply:
so "any"? but `el` is what *all* the "run cico" stages are tied to, so maybe we should clean them up once in a follow up?

Reply:
That is indeed what I was thinking about.

Reply:
updated